frontend/packages/shipwright-plugin/src/utils.ts

```diff
@@ -0,0 +1,73 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { getBuildRunStatusProps } from './components/buildrun-status/BuildRunStatus';
+import { BUILDRUN_TO_RESOURCE_MAP_LABEL } from './const';
+import { BuildRun, ComputedBuildRunStatus as runStatus } from './types';
+
+export type LatestBuildRunStatus = {
+  latestBuildRun: BuildRun;
+  status: string;
```

Comment:
Isn't this a ComputedBuildRunStatus?

```suggestion
  status: ComputedBuildRunStatus;
```

Reply:
In the code, if we set the status type as `ComputedBuildRunStatus`, then it is giving **type error** since the `getBuildRunStatusProps` function returns the type **StatusProps** which cannot be casted into **ComputedBuildRunStatus**.   

Also, if we do the opposite, i.e. status type as `StatusProps`, then the `getLatestBuildRunStatusforDeployment` is throwing type error since it returns the status as a string.

Therefore to solve this issue, it's better to keep the type as a **string** as it is accepted by both the functions.

Reply:
You can just use `getBuildRunStatus` instead of `getBuildRunStatusProps`, its returns a `ComputedBuildRunStatus`